explorer/models/azureRegistryNodes.ts

```diff
@@ -117,7 +127,7 @@ export class AzureRepositoryNode extends NodeBase {
     constructor(
         public readonly label: string,
         public readonly contextValue: string,
-        public readonly iconPath: { light: string | vscode.Uri; dark: string | vscode.Uri } = {
+        public readonly iconPath: AzureRegistryNode["iconPath"] = {
             light: path.join(__filename, '..', '..', '..', '..', 'images', 'light', 'Repository_16x.svg'),
```

Comment:
It seems like both are the same icon? Make sure that this changes when we get the real one